bin/VolcanoPlots_script.R

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env Rscript
+
+# Import library
```

Comment:
consider adding `library("conflicted")`. I recommend always importing it and it has already saved me quite some headache due to conflicting function names between dplyr and other packages.